src/mempools.c

```diff
@@ -365,9 +365,10 @@ struct memblock_s {
     } u;
 };
 
-#define MEMBLOCK_LIMIT (128)
```

Comment:
slaballoc and smalloc use (SMALL_BLOCK_NUM + 1) * sizeof(void*) as their limit. It's 68 bytes on 32-bit-plattforms, 136 bytes and 64-bit-plattforms. sysmalloc has no notion of small blocks. I guess that's why there is no global define for this constant (because it's allocator implementation specific).

We could move the SMALL_BLOCK_NUM define to xalloc.h, so mempool.c can use that instead of the hardcoded (16).

(Btw. in github it looks like you commented the line that was removed, not the one of the patch.)